spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -413,6 +413,7 @@
 "Zlib/Libpng License": zlib-acknowledgement
 "Zope Public License": ZPL-2.1
 "Zope Public": ZPL-2.1
+"['MIT']": MIT
```

Comment:
Maybe one thing to keep in mind for the future: If we see more misuses of setuptools' [license field](https://packaging.python.org/guides/distributing-packages-using-setuptools/#license) like this, we could consider parsing this as an array and adding the individual licenses as a work-around.